app/src/main/jni/download_ffmpeg.sh

```diff
@@ -23,10 +23,8 @@
 # along with MythTV-leanfront.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-scriptname=`readlink -e "$0"`
-scriptpath=`dirname "$scriptname"`
-scriptname=`basename "$scriptname" .sh`
+scriptpath="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
 cd "$scriptpath"
 
-(git -C ffmpeg pull || git clone git@github.com:FFmpeg/FFmpeg.git ffmpeg)
+(git -C ffmpeg pull || git clone git://source.ffmpeg.org/ffmpeg ffmpeg)
```

Comment:
Why change from github.com to ffmpeg.org?

Reply:
ffmpeg.org is the source, github is only a mirror. 
In the doc of exoplayer (https://github.com/google/ExoPlayer/blob/release-v2/extensions/ffmpeg/README.md), it shows `git://source.ffmpeg.org/ffmpeg`.